### PR TITLE
Fix launcher hf

### PR DESCRIPTION
**Pull request by robbiecarlton**
Is used by: https://github.com/Holo-Host/holofuel-rsm-ui/pull/209

### CODE REVIEW

File: src/stores/useHolochainStore.js [hunk 1]
```diff
@@ -9,28 +9,34 @@ import { hposHolochainCall } from '../services/hpos'
 
 const HC_APP_TIMEOUT = 35_000
 
+const __HC_LAUNCHER_ENV__ = "__HC_LAUNCHER_ENV__";
+const isLauncher = () => globalThis.window && __HC_LAUNCHER_ENV__ in globalThis.window;
+const getLauncherEnvironment = () => isLauncher() ? globalThis.window[__HC_LAUNCHER_ENV__] : undefined;
```
Comment (JettTech) [inline src/stores/useHolochainStore.js]:
```suggestion
```
Do we still need `getLauncherEnvironment`?  It seems like `isLauncher` is sufficient here now.

File: src/stores/useHolochainStore.js [hunk 2]
```diff
@@ -9,28 +9,34 @@ import { hposHolochainCall } from '../services/hpos'
...
     // BEGIN useInterfaceStore methods
 
     async initialize() {
       try {
-        const holochainClient = await AppWebsocket.connect(
-          app_ws_url,
-          HC_APP_TIMEOUT,
-          signal => useSignalStore().handleSignal(presentHcSignal(signal))
-        )
+        const holochainClient = await AppWebsocket.connect({
```
Comment (JettTech) [inline src/stores/useHolochainStore.js:32]: I don't remember where we landed on this?  Are we still supporting the pure hc case?  

If so, we will need a clause to check if the env is in launcher here and if not, then make the call to fetch the token and send it in with the app connection call too.

File: src/stores/useHolochainStore.js [hunk 3]
```diff
@@ -98,8 +102,7 @@ const makeUseHolochainStore = ({ installed_app_id, app_ws_url, hc_admin_port })
         throw new Error(`Couldn't find provisioned cell with role_name ${role_name}`)
       }
 
-      if( !this.signingCredentials)
-      {
+      if(!this.isLauncher && !this.signingCredentials) {
         this.setCredentials(cellId)
       }
```
Comment (JettTech) [inline src/stores/useHolochainStore.js]: ...and if we're not supporting the pure holochain case, then we can probably just remove this case and related method.

File: src/stores/useHolochainStore.js (hunk 2 above)
Comment (robbiecarlton) [reply src/stores/useHolochainStore.js:32]: We decided to not support native HC. I've removed all the relevant code

**Review by JettTech — APPROVED** [review]

**Pull request merged**